injection/sharedmain/main.go

```diff
@@ -118,100 +112,55 @@ func GetLeaderElectionConfig(ctx context.Context) (*kle.Config, error) {
 	return kle.NewConfigFromConfigMap(leaderElectionConfigMap)
```

Comment:
nit, since there's no longer an import conflict, maybe kle -> leaderelection and drop the kle from the import?

Reply:
Done